MSAL/src/public/MSALInteractiveTokenParameters.h

```diff
@@ -68,6 +70,9 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (instancetype)initWithScopes:(NSArray<NSString *> *)scopes NS_DESIGNATED_INITIALIZER;
 
+@property (nonatomic, nullable) WKWebView *customWebview;
+@property (nonatomic) MSALWebviewType webviewType;
```

Comment:
Does this mean you can change webview type per acquire token request?

Reply:
Only if acquireTokenWithParameters: is being used. InteractiveParameter will use the default value of global config at init that can be overwritten.